libraries/GCS_MAVLink/GCS_MAVLink.h

```diff
@@ -18,8 +18,8 @@
 // into progmem
 #define MAVLINK_MESSAGE_CRC(msgid) mavlink_get_message_crc(msgid)
 
-// allow four telemetry ports
-#define MAVLINK_COMM_NUM_BUFFERS 4
+// allow six telemetry ports
+#define MAVLINK_COMM_NUM_BUFFERS 5
```

Comment:
This is confusing for me. Why does the comment says 6 telemetry ports but the buffer is 5?